run_nose.py

```diff
@@ -0,0 +1,14 @@
+import re
+import sys
+from nose import run_exit
+
+if __name__ == '__main__':
+    try:
+        from twisted.internet import glib2reactor as reactor
+    except ImportError:
+        from twisted.internet import selectreactor as reactor
+
+    reactor.install()
+
+    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
```

Comment:
Why is this here? Please add some comments.

Reply:
umm, what's unclear?

Reply:
Oh, that's the old comment.